tools/bigquery-zos-mainframe-connector/USING_WITH_BQ.md

```diff
@@ -0,0 +1,227 @@
+  --skip_leading_rows <value>
+                           An integer that specifies the number of rows to skip at the beginning of the source file.
+  --source_format <value>  The format of the source data. Possible values include: CSV NEWLINE_DELIMITED_JSON AVRO DATASTORE_BACKUP PARQUET ORC (Default: ORC)
+  --time_partitioning_expiration <value>
+                           An integer that specifies (in seconds) when a time-based partition should be deleted. The expiration time evaluates to the partition's UTC date plus the integer value. A negative number indicates no expiration.
+  --time_partitioning_field <value>
+                           The field used to determine how to create a time-based partition. If time-based partitioning is enabled without this value, the table is partitioned based on the load time.
+  --time_partitioning_type <value>
+                           Enables time-based partitioning on a table and sets the partition type. Currently, the only possible value is DAY which generates one partition per day.
+  --require_partition_filter <value>
+                           If specified, a partition filter is required for queries over the supplied table. This flag can only be used with a partitioned table.
+  --use_avro_logical_types <value>
+                           If sourceFormat is set to AVRO, indicates whether to convert logical types into their corresponding types (such as TIMESTAMP) instead of only using their raw types (such as INTEGER).
+  --dataset_id <value>     The default dataset to use for requests. This flag is ignored when not applicable. You can set the value to [PROJECT_ID]:[DATASET] or [DATASET]. If [PROJECT_ID] is missing, the default project is used. You can override this setting by specifying the --project_id flag. The default value is ''.
+  --debug_mode <value>     Set logging level to debug. The default value is false.
+  --job_id <value>         The unique job ID to use for the request. If not specified in a job creation request, a job ID is generated. This flag applies only to commands that create jobs: cp, extract, load, and query.
+  --location <value>       A string corresponding to your region or multi-region location.
+  --project_id <value>     The project ID to use for requests. The default value is ''.
+  --synchronous_mode <value>
+                           If set to true, wait for the command to complete before returning, and use the job completion status as the error code. If set to false, the job is created, and successful completion status is used for the error code. The default value is true.
+  --sync <value>           If set to true, wait for the command to complete before returning, and use the job completion status as the error code. If set to false, the job is created, and successful completion status is used for the error code. The default value is true.
+  --stats_table <value>    tablespec of table to insert stats
+```
+
+
+
+## bq mk
+
```

Comment:
add a note stating that this is most commonly used to create external tables or native tables that need partitioning and clustering to be set up

Reply:
added